bylaws/001-mission-and-structure.md

```diff
@@ -71,16 +71,62 @@ Also, whilst not full members themselves, they have the ability to start any vot
+
+Should a Working Group be missing an Editor for 60 days; be missing a Sponsor for 60 days; have insufficient active members for 60 days; or show no signs of activity for six months, then the Core Committee may hold a Decision Vote to name a new Editor or Sponsor, following a suitable nomination process.  One of the options in that Decision Vote must be to dissolve the Working Group.  If no suitable candidates for Editor or Sponsor may be found, then the Working Group is automatically dissolved.
+
+If a Working Group is dissolved, a new Working Group covering the same scope may be formed at a later date through a new Entrance Vote.
+
+The Secretaries will ensure that the Working Group is provided with necessary resources to support their efforts, such as a dedicated GitHub repository, mailing list, chat room, or similar such tools.
+
+## Maintainers
+
+The Core Committee is ultimately responsible for all artifacts produced by the PHP FIG.  However, in the case of artifacts that are intended to change over time, the Core Committee may delegate that responsibility to a Maintainer for one or more related artifacts.
+
+Evolving artifacts follow Semantic Versioning.  If it is unclear if a given change would qualify as a bugfix vs a minor release, the Maintainer will assume minor release.
+
+* For changes that would qualify as bugfix releases, the Maintainer may issue a new release of the artifact at any time.
+* For changes that would qualify as minor releases, the Maintainer must notify the Core Committee via a post to the mailing list of an Intent to Merge a given change.  The Intent to Merge is subject to Implicit Approval with an Approval Vote if necessary.
+* For changes that would qualify as major releases, the release is subject to a mandatory Approval Vote from the Core Committee.
+
+The Editor of a Working Group is automatically the Maintainer of that Working Group's effort and output.
+
+The Maintainer of a given artifact is appointed by the Core Committee by Approval Vote.  Once a given artifact is created, the Maintainer may step down and name a replacement Maintainer at any time with Implicit Approval.  If necessary, a Decision vote to appoint a new Maintainer may be held following a suitable nomination period.
+
+The Secretaries will ensure that the Maintainer has the necessary access and resources to develop the artifact, such as access to a GitHub repository, mailing list, chat room, or similar such tools.
+
+## Artifacts
+
+The PHP FIG produces three primary forms of output artifact that are subject to established workflow processes.  This list is not exhaustive of the activity or publication that the PHP FIG may engage in.
```

Comment:
Please use the dashed form in any place in our documents:
```suggestion
The PHP-FIG produces three primary forms of output artifact that are subject to established workflow processes.  This list is not exhaustive of the activity or publication that the PHP-FIG may engage in.
```

Reply:
Curiously, the bylaws seem to use PHP FIG almost exclusively, not PHP-FIG.  (It's 15 to 0 in the mission-and-structure bylaw right now.)  That may be worth changing, but seems separate from the other changes here.

Reply:
If that's the case, it's a task for us @php-fig/secretaries, with no urgency.